decomp/semantics/uds.py

```diff
@@ -1093,7 +1058,10 @@ def __getitem__(self, k):
         return node_attrs, edge_attrs
 
 class NormalizedUDSDataset(UDSAnnotation):
-    """A Universal Decompositional Semantics dataset
+    """A normalized Universal Decompositional Semantics dataset
+
+       Attributes in a NormalizedUDSDataset may have only a single
```

Comment:
Expanded descriptions align with the quotes (see [docstring-standard](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard))